pkg/apis/application/v1alpha1/types.go

```diff
@@ -541,6 +546,26 @@ func (spec ApplicationSpec) GetProject() string {
 	return spec.Project
 }
 
+func (proj AppProject) IsDefault() bool {
+	return proj.Name == "" || proj.Name == common.DefaultAppProjectName
+}
+
+func (proj AppProject) IsClusterGroupKindPermitted(gk metav1.GroupKind) bool {
+	if proj.IsDefault() {
+		return true
+	}
+	for _, item := range proj.Spec.ClusterResources {
+		ok, err := filepath.Match(item.Kind, gk.Kind)
```

Comment:
Does this support `*` notation? If not I think we should support this.

Reply:
Yes, it is supported by `filepath.Match` function.